world/petitions/models.py

```diff
@@ -167,15 +173,41 @@ def display(self):
         """Gets string display of the amount purchased and by whom"""
         return "{} bought {}".format(self.buyer, self.amount)
 
+class PetitionSettings(SharedMemoryModel):
+    owner = models.ForeignKey("dominion.PlayerOrNpc", related_name="petition_settings")
```

Comment:
Probably a OneToOne with how you're using it, or move the fields into PlayerOrNpc as a whole. We'll probably make a more broad character-based settings object that'd absorb this at some point

Reply:
Yeah, this object exists just to make the settings modular and easy to clean up.